__tests__/app.test.js

```diff
@@ -0,0 +1,43 @@
+const request = require("supertest");
+const app = require("../app");
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+
+afterAll(() => {
+  if (db.end) db.end();
```

Comment:
don't need the conditional here, just 'return db.end()' will be fine

Reply:
Thanks - updated as suggested